collections/misc/collbackup.php

```diff
@@ -28,34 +32,34 @@ function submitBuForm(f){
 			return true;
     	}
     </script>
+    <style>
+    	fieldset{ padding:15px;width:350px }
+    	legend{ font-weight: bold }
+    	label{ font-weight: bold }
+    	#workingdiv{ float:left; margin-left:15px; font-weight: bold; }
+    </style>
 </head>
 <body>
-	<!-- This is inner text! -->
 	<div id="innertext">
 		<?php
 		if($isEditor){
 			?>
 			<form name="buform" action="../download/downloadhandler.php" method="post" onsubmit="return submitBuForm(this);">
-				<fieldset style="padding:15px;width:350px">
-					<legend><?php echo (isset($LANG['DWN_MOD'])?$LANG['DWN_MOD']:'Download Module'); ?></legend>
-					<div style="float:left;">
-						<?php echo (isset($LANG['DATA_SET'])?$LANG['DATA_SET']:'Data Set'); ?>:
-					</div>
-					<div style="float:left;height:50px">
-						<?php
-						//$cSet = str_replace('-','',strtolower($CHARSET));
-						?>
-						<input type="radio" name="cset" value="iso-8859-1" <?php echo (!$cSet || $cSet=='iso88591'?'checked':''); ?> /> ISO-8859-1 (western)<br/>
-						<input type="radio" name="cset" value="utf-8" <?php echo ($cSet=='utf8'?'checked':''); ?> /> UTF-8 (unicode)
+				<fieldset>
+					<legend><?= $LANG['DOWNLOAD_MODULE'] ?></legend>
+					<label><?= $LANG['ENCODING'] ?></label>:
```

Comment:
associate the label with the input by adding a `for` attribute to label and point it to the name of the `input` radio button tags, which I think should all get the same name. Their `id`s should be unique, though, right?

Reply:
I had thought that we could jus provide one label with the `for` attribute associated with the matching `name` attribute of each of th inputs. But after testing this out with Total Validator Pro, I'm seeing that it wants a label for EACh input element. I'm seeing that this is what has been implemented on other pages already. Here's my recommendation:


`<b><?= $LANG['ENCODING'] ?></b>:
					<div style="height:50px; margin: 10px">
						<input type="radio" id="cset-1" name="cset" value="iso-8859-1" <?= (!$cSet || $cSet == 'iso88591' ? 'checked' : ''); ?> /> <label for="cset-1">ISO-8859-1 (western)</label><br/>
						<input type="radio" id="cset-2" name="cset" value="utf-8" <?= ($cSet == 'utf8' ? 'checked' : ''); ?> /> <label for="cset-2">UTF-8 (unicode)</label>
					</div>`
					
In other words, change the $LANG['ENCODING'] to a `<b>` element and add unique `id`s to each `input` element and then give each their own `label` tag.

Alternatively, either remove `<?= $LANG['DOWNLOAD_MODULE'] ?>` as the legend and replace with $LANG['ENCODING'] or combine the two.